src/main/java/org/generationcp/middleware/api/germplasm/search/GermplasmSearchResponse.java

```diff
@@ -0,0 +1,290 @@
+package org.generationcp.middleware.api.germplasm.search;
+
+import org.generationcp.middleware.domain.inventory.GermplasmInventory;
+import org.generationcp.middleware.pojos.Germplasm;
+import org.pojomatic.Pojomatic;
+import org.pojomatic.annotations.AutoProperty;
+
+import java.util.Map;
+import java.util.Objects;
+
+@AutoProperty
+public class GermplasmSearchResponse {
+
+	// canonical columns
+
+	private int gid;
+	private Integer groupId;
+	private String names;
+	private String methodName;
+	private String locationName;
+	private String stockIds;
```

Comment:
@nahuel-soldevilla Let's check the inventory fields with @mcrimi  
We are planning to remove stockIds in the current germplasm search and availableBalance was splitted in 2 columns in other places: availableBalance and unit. For the inventory, I think that the lotCount should be included here too. I will merge it anyway, it can be done in the next sprint